### ISSUE TITLE
Could you please release the OpenPoseLightning model file, thanks!

### ISSUE THREAD

**Issue by xudif** [rongtuech/handbodypose#1]
Hello, I need OpenPoseLightning model, but I run the train.py file for network training, and it cannot achieve good results. So I hope you can upload the OpenPoseLightning model file, thank you so much!

**Comment by rongtuech** [comment]
Because of my school policy, I cannot make it public. If it is hard to get the acceptance result on your data, please use the pre-trained model https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch#pre-trained-model,  and re-train on your dataset.

**Comment by rongtuech** [comment]
Furthermore, please use the inputs as the same size as training inputs when reference. If not, the ACC may be decreased.

**Comment by xudif** [comment]
Thanks for your help.
I found that pre-trained model is expects normalized image (mean=[128, 128, 128], scale=[1/256, 1/256, 1/256]) in planar BGR format and .
Do I need to add normalization in COCOTransformation class?

**Comment by xudif** [comment]
I tried to use the pre-trained model directly in image_inference.py, but the hand in the generated images could not be accurately identified. May I ask what caused it?

**Comment by rongtuech** [comment]
the dataset.py code already done for you. Furthermore, if you write your own dataset code, you can use torchvision.transform.toTensor() function, it will do the same.

**Comment by rongtuech** [comment]
> I tried to use the pre-trained model directly in image_inference.py, but the hand in the generated images could not be accurately identified. May I ask what caused it?

it is better to be retrained.
About the accuracy of hand -> you mean bounding box of hands or ???

**Comment by xudif** [comment]
> > I tried to use the pre-trained model directly in image_inference.py, but the hand in the generated images could not be accurately identified. May I ask what caused it?
> 
> it is better to be retrained.
> About the accuracy of hand -> you mean bounding box of hands or ???

The hand in the image cannot identify and produce key points and bounding box of hands.

**Comment by xudif** [comment]
like this [https://github.com/xudif/Multi-focus-Image-Fusion-Dataset/blob/4d6fabd1a7407e19aa4967ec089766f405a51999/img_with_pose_5.jpg](url)

**Comment by rongtuech** [comment]
> The hand in the image cannot identify and produce key points and bounding box of hands.

The hand bounding box is defined by wrist joints. Therefore, if the model is too bad to recognize those joints -> can not get hand keypoints.

**Comment by xudif** [comment]
Thanks, I will try to retrain the model.
